Week-9/demo-react-app/src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
```

Comment:
Try to add more test cases.. it will be good learning for u and try to add test cases for individual component